src/actions/heap/heap.ts

```diff
@@ -129,7 +123,7 @@ export class HeapAction extends Hub.Action {
       logger.error("Error in preflight resolvers", { ...logTag, err })
       return new Hub.ActionResponse({
         success: false,
-        message: getErrorMessage(err),
+        message: err.message,
```

Comment:
Just incase 
```suggestion
        message: err?.message,
```

Reply:
tried that before, the lint rule will reject it